scripts/bootstrap-osx.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+# used in travis to download & install libsodium
+
+brew update
+brew install libsodium
```

Comment:
We don't need it

Reply:
= note: /usr/bin/ld: cannot find -lsodium
          collect2: error: ld returned 1 exit status

Reply:
Ah sorry. I did not see you enabled osx builds